### ISSUE TITLE
tpl: Add intersect operator to where

### ISSUE THREAD

**Issue by moorereason** [spf13/hugo#2029]
Add `intersect` as an operator for `where`:

```
{{ range where .Site.Data.person "roles" "intersect" .Site.Params.admin_roles }}
```
If `len(intersect(l1, l2)) > 0`, add the element to the `where` results.

**Comment by bep** [comment]
https://github.com/spf13/hugo/pull/1945

**Comment by moorereason** [comment]
Fixed by #1945
